### DISCUSSION TITLE
Examples is missing

### DISCUSSION THREAD

**Discussion by gnumoksha** [dompdf/dompdf#1736] [General]
From: https://dompdf.net/examples.php:

> Below are some sample files. The PDF version is generated on the fly by dompdf. (**The source HTML & CSS for these files is included in the test/ directory of the distribution package**.)

But I'm unable to find "test" directory in this project.

**Comment by kirame09** [comment]
+1 for this.

**Comment by kirame09** [comment]
@gnumoksha , Here is the link https://github.com/dompdf/dompdf/tree/tagged-pdf/www/test

**Comment by khanks** [comment]
... Not ...

> 
> 
> @gnumoksha , Here is the link https://github.com/dompdf/dompdf/tree/tagged-pdf/www/test

... Nope, not there...

**Comment by bsweeney** [comment]
Examples were removed from the project with the 0.7.0 release. We may resurrect those at some point in a separate project. In the meantime you can get to the samples that were included in past released by browsing older versions of the code. Here's a link to the samples from the 0.6.2 release: https://github.com/dompdf/dompdf/tree/v0.6.2/www/test
